docs/ci-guides/github-actions.mdx

```diff
@@ -97,3 +97,21 @@ jobs:
 ```
 
 </small>
+
+### Why are there elements in my screenshots that are not visible in my local code?
+
+When running tests on GitHub, it defaults to performing tests on the merge commit. This means that it behaves as if a rebase had been done before running the tests. As a result, some elements may appear in your screenshots that are not visible in your local code.
+
+It is not a problem with Argos because it is handled automatically by our system but it can be suprising.
+
+If you wish to disable this behavior and have GitHub check out the pull request's HEAD commit instead of the merge commit, you can use the following configuration in your workflow file:
+
+```yml
+- uses: actions/checkout@v3
+  with:
+    ref: ${{ github.event.pull_request.head.sha }}
+```
+
+By specifying the `ref` parameter with the value `${{ github.event.pull_request.head.sha }}`, you ensure that GitHub checks out the specific commit associated with the pull request, allowing your screenshots to accurately reflect the code changes made in that commit.
+
+Make sure to update your workflow file with this configuration if you want to avoid including extraneous elements in your screenshots.
```

Comment:
### Why do my screenshots show elements not present in my local code?

When you run tests on GitHub, it defaults to testing the version of your code that would exist after a merge. This is why you might see some unexpected elements in your screenshots - they're not in your local code, but they would be in the code after a merge.

Argos is designed to handle this situation automatically, but it can catch you by surprise.

If you prefer to run tests on the exact code you've written, without any potential merge changes, you can adjust your workflow file on GitHub. Add the following configuration:

  ```yml
  - uses: actions/checkout@v3
    with:
      ref: ${{ github.event.pull_request.head.sha }}
  ```

This setting tells GitHub to test the exact code from your most recent commit (the "HEAD" commit) in the pull request. This way, your screenshots will reflect only the changes you've made in that commit.

Remember to update your workflow file with this setting if you'd prefer to avoid any surprise elements in your screenshots.